yapapi/payload/vm.py

```diff
@@ -204,7 +205,7 @@ async def repo(
     )
 
 
-def resolve_repo_srv(repo_srv, fallback_url=_FALLBACK_REPO_URL) -> str:
+def resolve_repo_srv(repo_srv: str, fallback_url=_FALLBACK_REPO_URL) -> str:
```

Comment:
ah, if the timeout used there is the `default`, I guess, there should be an argument to give a non-default

Reply:
done